GW2EIEvtcParser/EIData/ProfHelpers/Engineer/HolosmithHelper.cs

```diff
@@ -42,6 +42,7 @@ public static bool IsPhotonForgeTransform(long id)
     [
         new BuffOnActorDamageModifier(Mod_LasersEdge, LasersEdge, "Laser's Edge", "15%", DamageSource.NoPets, 15.0, DamageType.Strike, DamageType.All, Source.Holosmith, ByPresence, TraitImages.LasersEdge, DamageModifierMode.PvE)
             .WithBuilds(GW2Builds.StartOfLife, GW2Builds.July2019Balance),
+        new DamageLogDamageModifier(Mod_SolarFocusingLens, "Solar Focusing Lens", "10%", DamageSource.NoPets, 10.0, DamageType.Strike, DamageType.All, Source.Holosmith, TraitImages.SolarFocusingLens, (x, log) => log.CombatData.HasLostBuff(Afterburner, x.From, x.Time, ServerDelayConstant), DamageModifierMode.All),
```

Comment:
Wouldn't server delay constant potentially cause more elements than expected to be picked up by this? Like 1 stack remaining, 2 damage events on server delay close time stamps, they would both qualify no?

Reply:
I've added it in case the damage is registered after the buff removal, I'll change it to whatever you think is best

Reply:
Flag is as approximate for now, we may need a special logic to handle it in a cleaner manner.